services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/mounted_fs.py

```diff
@@ -36,11 +36,16 @@ class MountedVolumes:
     """
 
     def __init__(
-        self, inputs_path: Path, outputs_path: Path, state_paths: List[Path]
+        self,
+        inputs_path: Path,
+        outputs_path: Path,
+        state_paths: List[Path],
+        state_exclude: List[str],
```

Comment:
MINOR: ``state_exclude`` is a list of  glob patterns to exclude, right? perhaps name ``state_exclude_globs`` ( notice that globs (``fnmatch``) differs from regex (``re``) patterns!)

Reply:
Actually it is a list of Unix shell-style wildcards coming from fnmatch. Which is neither of them.

Reply:
I have seen those referred as ``glob``s. Check https://docs.python.org/3/library/fnmatch.html

Reply:
My mistake if I did that. The ones used here are unix shell-style ones.